### PR TITLE
feat: add entityRef to Entity Inspector view

### PR DESCRIPTION
**Pull request by clanghout**
## Hey, I just made a Pull Request!

Occasionally, I would like to get an entity ref, but the UI has no place to copy it from. The Entity Inspector seems like a nice place for it. 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
image

new screenshot with copy buttons:
image

### CODE REVIEW

File: plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx [hunk 1]
```diff
@@ -90,6 +91,12 @@ export function OverviewPage(props: { entity: AlphaEntity }) {
                 <ListItemText primary="etag" secondary={metadata.etag} />
               </ListItem>
             )}
+            <ListItem>
+              <ListItemText
+                primary="entityRef"
+                secondary={stringifyEntityRef(props.entity)}
```
Comment (freben) [inline plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx:97]: Hey this is great, while we're here coooould I perhaps ask nicely for a copy button for the value too? :D

You can use a [`CopyTextButton`](https://github.com/backstage/backstage/blob/29716aa3039ec05358e26fcedd4e437673f3dd74/packages/core-components/src/components/CopyTextButton/CopyTextButton.tsx#L80) as a [`ListItemSecondaryAction`](https://v4.mui.com/components/lists/#interactive) for example 🙏

**Review by freben — APPROVED** [review]
Optional ask for one more feature, preapproving anyway! Over to the catalog maintainers

File: plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx (hunk 1 above)
Comment (clanghout) [reply plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx:97]: will give it a look later today :D was thinking about that as well.

Do you think any other fields could use this same functionality?

Comment (freben) [reply plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx:97]: Sure!

**Review by alde — APPROVED** [review]

File: plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx (hunk 1 above)
Comment (clanghout) [reply plugins/catalog-react/src/components/InspectEntityDialog/components/OverviewPage.tsx]: done for etag, uid and entityRef. In my initial thoughts apiversion, entitykind and spec.type don't benefit much from having a copy button. If you feel like I should add it for those too for consistency, I'd be happy to add it there too.

**Review by freben — APPROVED** [review]
That's great!

**Pull request merged**